install_config/configuring_authentication.adoc

```diff
@@ -1391,3 +1391,47 @@ a secret with 32 or 64 bytes.
 <3> Encrypting secret, used to encrypt sessions. Must be 16, 24, or 32
 characters long, to select AES-128, AES-192, or AES-256.
 ====
+
+[[configuring-authentication-security]]
+== Security
+
+Administrators can prevent clients from accessing the API by their `User-Agent`
+header in the new `*userAgentMatching*` configuration setting.
+
+For example, you can deny clients using particular libraries or binaries. This
+example denies the Kubernetes 1.2 client binary, OpenShift Origin 1.1.3
+binary, and the POST and PUT *httpVerbs*:
+
+====
+----
+policyConfig:
+  userAgentMatchingConfig:
+    defaultRejectionMessage: "Your client is too old.  Go to https://example.org to update it."
+    deniedClients:
+    - regex: '\w+/v(?:(?:1\.1\.1)|(?:1\.0\.1)) \(.+/.+\) openshift/\w{7}'
+    - regex: '\w+/v(?:1\.1\.3) \(.+/.+\) openshift/\w{7}'
+    - regex: '\w+/v1\.2\.0 \(.+/.+\) kubernetes/\w{7}'amp`
```

Comment:
BTW, what does "amp\`" mean here? (I am not familiar with adoc syntax. If it is intended, ignore this comment)